apps/src/craft/code-connection/craft.js

```diff
@@ -215,6 +215,9 @@ export const executeUserCode = function (client, code) {
     },
     createBlockPos: function (x, y, z, prefix) {
       return encodeURIComponent(`${prefix}${x} ${prefix}${y} ${prefix}${z}`);
+    },
+    getVariable: function (value) {
+      return `${value}`;
```

Comment:
I was following the same protocol for the accepting of variables as arguments, but these blocks didn't expect the same string formatting the others did (as createBlockPos() returns a encodeURIComponent). I wanted to make sure that we could grab single variables for blocks that aren't looking at coordinates, such as:
![screen_shot_2017-06-21_at_1 54 14_pm](https://user-images.githubusercontent.com/28878212/27406371-7a4b044a-5689-11e7-9974-7bd6a9178bce.png)

If there is a better way to do this, or if I made it more convoluted than it had to be, then I can totally make changes. I was simply trying to avoid the situation I had previously where passing in a variable wouldn't be read properly since the face value of the input was no longer explicitly a number and might change during runtime.